grpc/src/test/java/com/linecorp/armeria/it/grpc/HttpJsonTranscodingTest.java

```diff
@@ -788,12 +788,11 @@ void shouldAcceptResponseBodyValueAnonymusField() throws JsonProcessingException
     }
 
     @Test
-    void shouldAcceptResponseBodyValueNoMatchInside() throws JsonProcessingException {
+    void shouldDenyMalformedJson() throws JsonProcessingException {
         final String jsonContent = "{\"value\":\"value\",\"structBody\":{\"structBody\":\"struct_value\"}";
```

Comment:
I actually think this is a typo where a `}` character at the end is missing.

```
final String jsonContent = "{\"value\":\"value\",\"structBody\":{\"structBody\":\"struct_value\"}}";
```
Do you mind reverting this test and creating a separate test case which tests for malformed json requests?

Reply:
Yes, my mistake, I misread the test. I thought it was explicitly testing the invalid json returning an empty body